src/NIDAQ.jl

```diff
@@ -33,14 +33,17 @@ export stop, clear
 
 # channels
 export analog_input, analog_output, digital_input, digital_output
+export acceleration_input
 export count_edges, quadrature_input, line_to_line, generate_pulses
 
 # properties
-export devices, channel_type, getproperties, setproperty!
-export analog_input_ranges, analog_output_ranges
-export analog_input_channels, analog_output_channels
-export digital_input_channels, digital_output_channels
-export counter_input_channels, counter_output_channels
+export devices, channel_type, getproperties #, setproperty!
+export analog_input_ranges,          analog_output_ranges
```

Comment:
maybe deprecate analog_{input,output}_ranges?

Reply:
I left them for backward compatibility. I don´t mind removing it...

Reply:
i mean to suggest to use `@deprecate` (see https://docs.julialang.org/en/v1/base/base/#Base.@deprecate).  definitely need to leave them in for backwards compatibility.